tests/playwright/tests/Transactions/Send usdt from token screen.spec.ts

```diff
@@ -20,16 +20,33 @@ test('test', async ({ page }) => {
     await page.getByRole('button', { name: 'Continue' }).click();
     await page.locator('#react-portal-modal-container').getByRole('textbox').fill('Ananas');
     await page.getByRole('button', { name: 'Save' }).click();
-    await page.locator('div:nth-child(10) > .sc-dOSSlk > .sc-isYNRO').click();
+});
+
+test.afterEach(async ({ page }) => {
+    await page.getByRole('link', { name: 'Settings' }).click();
+    await page.goto(`/wallet-settings`);
```

Comment:
Please note that it have issues during run, timeout of 30000ms exceeded
I guess it's because of reloading page

```
await page.getByRole('link', { name: 'Settings' }).click();
await page.goto(`/wallet-settings`);
```

This 2 lines do exactly the same, so better to remove second line and do not reload the page.